swap_meet/vendor.py

```diff
@@ -1,2 +1,98 @@
 class Vendor:
-    pass
\ No newline at end of file
+    MAX_AGE = 1000
+# Wave 1    
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory 
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+    def remove(self,item):
+        for inventory_item in self.inventory:
+            if inventory_item == item:
+                self.inventory.remove(item)
+                return item
+        return False
+
+# Wave 2
+    def get_by_category(self, category):
+        item_list = []
+        for item in self.inventory:
+            if item.category == category:
+                item_list.append(item)
+        return item_list
```

Comment:
Really nicely done! As an alternative, can you consider how this could be done using a list comprehension? Which method do you prefer?